src/Language/PureScript/Docs/Convert/Single.hs

```diff
@@ -128,7 +128,7 @@ mkDeclaration (ss, com) title info =
               , declSourceSpan = Just ss -- TODO: make this non-optional when we next break the format
               , declChildren   = []
               , declInfo       = info
-              , declKind       = Nothing -- kind sigs are added in augment pass
+              , declKind       = Nothing -- explicit kind signatures are added in augment pass
```

Comment:
The treatment of explicit versus inferred kind signatures is irrelevant to this piece of code, so I think it shouldn’t care about the distinction - especially because when we do update this to include inferred kind signatures, it’s going to be very easy to forget to update this comment, because the code in this function isn’t going to have to change. I’d suggest leaving this comment as it was before.

Reply:
Ok. I've removed the commit that made that change.